src/converter/Main.java

```diff
@@ -25,6 +25,7 @@ public static void main(String[] args) {
         System.out.println();
         System.out.println("Available converters:");
         System.out.println("  FlameGraph input.collapsed output.html");
+        System.out.println("  Flamegraph --diff input1.collapsed input2.collapsed output.html");
```

Comment:
I hope you don't mind the drive-by review. I was just testing this functionality and encountered this.


```suggestion
        System.out.println("  FlameGraph --diff input1.collapsed input2.collapsed output.html");
```

Lowercased `G` would result in an error: `Error: Could not find or load main class Flamegraph`

Reply:
Not at all, thanks for catching it!